### ISSUE TITLE
k-table styles from kendo-jquery are missing from the theme

### ISSUE THREAD

**Issue by valchev** [telerik/kendo-themes#856]
## I'm submitting a...

* Bug report 

## Current behavior

`.k-table` styles (and probably others too) are appended directly to the Kendo jQuery UI Editor's iframe and are missing from the theme. 

## Expected behavior

Add the corresponding styles to the theme, so that the other suites, that support non-iframe rendering mode, can use them.

## What is the motivation or use case for changing the behavior?

Reusing the current styles.

**Comment by nstoychev** [comment]
@joneff - Here are the styles loaded in the editor's iframe. Can we add them to the themes?
```css
.k-content table {
   margin: 0;
   border-collapse: collapse;
   table-layout: fixed;
   width: 100%;
   overflow: hidden;
}

.k-content td,
.k-content th {
   min-width: 1em;
   border: 1px solid #ddd;
   padding: 3px 5px;
   vertical-align: top;
   box-sizing: border-box;
   position: relative;
}

.k-content th {
   font-weight: bold;
   text-align: left;
}

/* Give selected cells a blue overlay */
/* https://github.com/ProseMirror/prosemirror-tables/blob/master/style/tables.css */
.k-content .selectedCell:after {
   z-index: 2;
   position: absolute;
   content: "";
   left: 0;
   right: 0;
   top: 0;
   bottom: 0;
   background: rgba(200, 200, 255, 0.4);
   pointer-events: none;
}
```

**Comment by joneff** [comment]
shipped with ae7b64322559cddf808db0f29c66b0d4db5d0b80
